src/services/currentWindow-service.js

```diff
@@ -25,6 +25,10 @@
                 $rootScope.$broadcast('updateFavourites', event.stock);
             });
 
+            window.addEventListener('windowClosed', () => {
+                $rootScope.$broadcast('windowClosed');
+            });
```

Comment:
This sort of thing is getting quite common place. Not in the scope of this PR but I think it's worth considering bringing the `window`/`$rootScope` event rebinding into the `currentWindowService` for a future release.